packages/modal/src/Modal.spec.tsx

```diff
@@ -58,6 +58,12 @@ describe('packages/modal', () => {
       window.dispatchEvent(new MouseEvent('click', { bubbles: true }));
       expect(container.innerHTML).toBe('');
     });
+
+    test('when "closeOnBackdropClick" is false', () => {
+      renderModal({ closeOnBackdropClick: false, open: true });
+      window.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+      expect(screen.getByText(modalContent)).toBeInTheDocument();
```

Comment:
I still don't like the pattern of selecting by text and other attributes, and would prefer a test-id. Technically, this only gets a child of the modal, and doesn't test that the whole modal is still in the document.

Reply:
Happy to switch this, just trying to follow the documented preferred patterns but if you feel strongly, let me know.

Reply:
I think I feel strongly enough about this – especially given that it doesn't technically test this correctly (as noted above).

Reply:
Fixed @DesignerDave!